config/sync/archipelago_subtheme.settings.yml

```diff
@@ -91,3 +91,5 @@ bootstrap_barrio_radio: custom
 bootstrap_barrio_checkbox: switch
 bootstrap_barrio_select: custom
 bootstrap_barrio_file: custom
+bootstrap_barrio_region_clean_editor_toolbar: 'test'
```

Comment:
This is wrong.

These last two lines are

bootstrap_barrio_region_clean_editor_toolbar: 0
bootstrap_barrio_region_class_editor_toolbar: row

Where does that 'test' comes from?

Reply:
@DiegoPino thank you for finding that. I pushed the fixed value here and in the subtheme repo.